test/source/tests/tests/account.ts

```diff
@@ -45,7 +45,9 @@ export const defineConsumerAcctTests = (testVariant: TestVariant, testWithNewBro
       fileInput = await composePage.target.$('input[type=file]');
       await fileInput!.uploadFile('test/samples/large.jpg');
       await Util.sleep(2); // give it a little time to make tests less brittle
-      await ComposePageRecipe.sendAndClose(composePage);
+      await composePage.waitAndClick('@action-send', { delay: 1 });
+      await composePage.page.waitForFunction(() => document.querySelector('[data-test="action-send"]')!.textContent!.includes('%'));
```

Comment:
This could end up being flaky - it would be better if we had a function like await `await frame.waitForSelectorWithText(`@selector`, 'some text')`.

Let's see how it goes.